java/src/main/java/vw/responses/ActionScores.java

```diff
@@ -0,0 +1,36 @@
+package vw.responses;
+
+import java.util.Arrays;
+
+/**
+ * Created by jmorra on 8/12/16.
+ */
+public class ActionScores {
+
+    private final ActionScore[] actionScores;
```

Comment:
I agree.  Writing the JNI for a `Collections.unmodifiableList` would be very cumbersome and I'd rather just add a comment.